github_connector/models/github_organization_serie.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class GithubOrganizationSerie(models.Model):
+    _name = 'github.organization.serie'
```

Comment:
What is an Organization Serie?

Reply:
@legalsylvain @elicoidal @lasley So it seams using the term milestone for that concept generates confusion (at least in my head). 

Does github have a concept of organization level milestone? I dont find it and github_connector does not synchronize it (it's populated from a data file). So maybe we could move that concept out of github_connector to github_connector_odoo and name that series? @legalsylvain does that make sense technically? (it might be too big a change at this stage)

Reply:
> @legalsylvain does that make sense technically? (it might be too big a change at this stage)

You mean, reverting the rename github.organization.milestone into github.organization.serie. or wathever. It's possible at this stage, not a huge work. I'd just like to finish this PR ! ;-)

Reply:
@sbidoul @legalsylvain 
I am ok with any name as long as it is consistent with github naming. I dont want to end up with Series in Odoo while it is Milestones in Github.
![screenshot from 2017-10-13 11-59-29](https://user-images.githubusercontent.com/7600613/31541138-006beb88-b00e-11e7-96ad-f035a84b6678.png)

Reply:
@elicoidal AFAICT github_connector does not synchronize github milestones. So my suggestion is to rename `github.organization.milestone` to `github.odoo.series` instead, and move it from github_connector to github_connector_odoo, so it's clear it's not a github concept but an odoo specific concept.

The fact that we have milestones named after odoo series is accidental and something that we may want to change in the future. For instance, in OCA/mis-builder I'm experimenting with a more [traditional use of milestones](https://github.com/OCA/mis-builder/milestone/2) to do release management.

Reply:
> So maybe we could move that concept out of github_connector to github_connector_odoo and name that series

I did'nt see that part, but this is not possible at this state. the current "serie / milestone / whatever" concept is here to know which repository odoo will download and analyse. download and analyse mechanism are in github_connector and not in github_connector_odoo. Other point it is not depending on odoo I think. Having multiple repositories, with series 1 / 2 / 3, and multiple branches name like 1.0_ADD_something is not odoo dependent.

In a few words github_connector objective : synchronize repositories (and social things like teams, contributors, etc... (and maybe tomorrow PR / issues / comments)
github_connector_odoo objective : analyze repositories IF repositories are containing odoo modules. (with manifest to analyze, etc...)

Reply:
@legalsylvain I understand. So it's the list of branches to sync in that organization. So definitely not milestones. I liked series. Or github.organization.branch but it might be confusing too.